pkg/kubectl/resource_printer.go

```diff
@@ -1396,7 +1396,10 @@ func printHorizontalPodAutoscaler(hpa *extensions.HorizontalPodAutoscaler, w io.
 	if hpa.Status.CurrentCPUUtilizationPercentage != nil {
 		current = fmt.Sprintf("%d%%", *hpa.Status.CurrentCPUUtilizationPercentage)
 	}
-	minPods := hpa.Spec.MinReplicas
+	minPods := "<unknown>"
```

Comment:
I think there is no difference what the default would be. It should never happen indeed.